src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,330 @@
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__center">
+                        <svg
+                            class="icon file-preview-card__general-icon"
+                            aria-hidden="true"
+                        >
+                            <use href="#icon-file-24"/>
+                        </svg>
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                    <div class="file-preview-card__label">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Preview"
+                        >
+                            <span class="file-preview-card__label-text">
+                                CSV
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__center">
+                <svg
+                    class="icon file-preview-card__general-icon"
+                    aria-hidden="true"
+                >
+                    <use href="#icon-file-24"/>
+                </svg>
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__label">
+                <button class="icon-btn" type="button" aria-label="Preview">
+                    <span class="file-preview-card__label-text">
+                        CSV
+                    </span>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Overflow</h3>
```

Comment:
What do we mean by overflow in this context? When I think of overflow I think of CSS overflow, so maybe we change the name, but we should also add a short sentence to describe the intended use of this variant. Is it intended to be used as the last element in a group when more elements exist? If so let's try and find a different name than overflow.

Reply:
Makes sense, I'll rename it to `See more`. Let me know if there is something more appropriate